x-pack/plugins/triggers_actions_ui/public/application/internal/shareable_components_sandbox/shareable_components_sandbox.tsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState } from 'react';
+import { getRuleStatusDropdownLazy } from '../../../common/get_rule_status_dropdown';
+
+export const InternalShareableComponentsSandbox: React.FC<{}> = () => {
```

Comment:
So for the components that I'm working on, I am rebasing off of this branch since I also need to E2E test the components. I'm wondering if we could maybe move the rule status dropdown rendering to its own file? so I can add my components in this sandbox as well

maybe something like?
```
return (
  <div>
    <RuleStatusDropdownSandbox />
    <OtherSandboxes />
  </div>
);
```

Reply:
Can do

Reply:
awesome!